project/Dependencies.scala

```diff
@@ -65,6 +65,7 @@ object Dependencies {
   )
 
   private val sprayServerDependencies = List(
+    "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
```

Comment:
Good catch. I'm guessing the provided was copy/pasta. Also should be using the value `swaggerUiVersion` (also edited in this PR) instead of `"2.2.2"`.